announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-delete.vm

```diff
@@ -15,8 +15,8 @@
 	<h3>
 		$tlang.getString("del.deleting")
 	</h3>
-	#if ($alertMessage)<div class="alertMessage">$tlang.getFormattedMessage("gen.alert.message", $alertMessage)</div>#end
-	<div class="alertMessage">$tlang.getString("del.areyou")</div>
+	#if ($alertMessage)<div class="sak-banner-error">$tlang.getFormattedMessage("gen.alert.message", $alertMessage)</div>#end
+	<div class="sak-banner-error">$tlang.getString("del.areyou")</div>
```

Comment:
This seems like a warning message instead of an error. The English message is "Are you sure you want to delete the following announcements?".

Reply:
changed line 19 to warn, leaving the banner on 18 as sak-banner-error. Thanks shawn!